test/ssl-tests/17-renegotiate.cnf.in

```diff
@@ -261,6 +261,37 @@ our @tests_tls1_2 = (
             "ResumptionExpected" => "No",
             "ExpectedResult" => "ServerFail"
         }
+    },
+    {
+        name => "allow-client-renegotiation",
+        server => {
+            "MaxProtocol" => "TLSv1.2",
+            "Options" => "ClientRenegotiation",
+        },
+        client => {
+            "MaxProtocol" => "TLSv1.2"
+        },
+        test => {
+            "Method" => "TLS",
+            "HandshakeMode" => "RenegotiateClient",
+            "ResumptionExpected" => "No",
+            "ExpectedResult" => "Success"
+        }
+    },
+    {
+        name => "no-client-renegotiation",
+        server => {
+            "MaxProtocol" => "TLSv1.2"
```

Comment:
Since `SSL_OP_ALLOW_CLIENT_RENEGOTIATION` is "on" by default for this test, you need to turn it off again, e.g.

````
            "Options" => "-ClientRenegotiation",
````